en/docs/catalogs/samples/api-policy.md

```diff
@@ -1,8 +1,6 @@
-# API Policy
+## API Policy Samples
 
-API policies are a powerful tool in WSO2 APK for enforcing business logic and modifying API requests and responses. You can attach policies at the gateway-level, API-level or operation-level, in either the request or response flow
+Refer [Manage API Policy](../../create-api/create-and-attach-api-policies/api-policies-overview.md) for more information on how to configure API Policy with API.
```

Comment:
Shall we fix this to `an API Policy with an API`?

Reply:
Fixed in PR #583